### DISCUSSION TITLE
Same config, different fill mode in magit commit

### DISCUSSION THREAD

**Discussion by hansbogert** [syl20bnr/spacemacs#16475] [Q&A] [answered]
I'm sharing my spacemacs config between a ubuntu machine and a macbook, so I expect a very similar feel on both machines. However when I open a commit message window (through magit) on the ubuntu machine, the fill-column is set to 120. Whereas on the macbook it's set to 80.  Other files have a fill-column of 80 on both machines, so it doesn't seem like a global which is different between machines.  

Can anyone help me debug where this difference may come from?

Both are on latest spacemacs develop

**Comment by bcc32** [comment, marked as answer]
Have you tried [`M-x debug-on-variable-change`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Variable-Debugging.html)?  It should help you pin down at least which package/function is setting it to 120.

**Reply by hansbogert** [reply]
Thanks, now I understand: this was due to a `.editorconfig` being present.

**Reply by bryce-carson** [reply]
> Thanks, now I understand: this was due to a `.editorconfig` being present.

Magit respects _.editorconfig_? Interesting. Could you write a PR adding documentation of that (and your understanding of the topic) to the Git layer? It'd be helpful to the community, likely.
